src/store/audio.ts

```diff
@@ -42,7 +42,10 @@ async function generateUniqueIdAndQuery(
   audioItem = JSON.parse(JSON.stringify(audioItem)) as AudioItem;
   const audioQuery = audioItem.query;
   if (audioQuery != undefined) {
-    audioQuery.outputSamplingRate = state.savingSetting.outputSamplingRate;
+    const outputSamplingRate = audioItem.engineId
+      ? state.engineSetting[audioItem.engineId].outputSamplingRate
+      : "engineDefault";
+    audioQuery.outputSamplingRate = outputSamplingRate;
```

Comment:
engineIdはundefinedになりえないはず＋デフォルトが`engineDefault`であるという処理がここにコピーされているので、throwのが適切かもです。
```suggestion
    if (audioItem.engineId == undefined)
      throw new Error(`audioItem.engineId is undefined`);
    audioQuery.outputSamplingRate =
      state.engineSetting[audioItem.engineId].outputSamplingRate;
```